src/mlte/measurement/measurement.py

```diff
@@ -36,9 +35,7 @@ def __init__(self, instance: Measurement, identifier: str):
         :param identifier: A unique identifier for the instance
         :type identifier: str
         """
-        self.metadata = MeasurementMetadata(
-            type(instance).__name__, Identifier(identifier)
-        )
+        self.metadata = MeasurementMetadata(type(instance).__name__, identifier)
```

Comment:
Inside MeasurementMetadata, it will be an identifier. But instead of passing an identifier as an argument, I am passing a string, and only interally it will create an Identifier based on this. To be honest, I don't recall exactly why I made this change but I can check. It made sense to simplify some of the depdencies. I think it was to avoid having Measurement depend on Identifier as well, and only make it depend on MeasurementMetadata.

Reply:
Resolved.